src/fe-common/core/completion.c

```diff
@@ -241,28 +241,37 @@ char *word_complete(WINDOW_REC *window, const char *line, int *pos, int erase, i
 	if (complist == NULL)
 		return NULL;
 
+	/* escape if the word doesn't begin with '/' and expand_escapes are turned on */
+	data = *line != '/' && settings_get_bool("expand_escapes") ?
```

Comment:
Are you trying to check whether `line` is a command?
If so you should honour the `cmdchars` option instead of checking for `/` only.